core/src/main/java/com/microsoft/applicationinsights/internal/channel/sampling/FixedRateTelemetrySampler.java

```diff
@@ -113,7 +113,7 @@ else if (!includeTypes.isEmpty() && !includeTypes.contains(telemetry.getClass())
 
                         return false;
                     }
-                    samplingSupportingTelemetry.setSamplingPercentage(telemetrySamplingScore);
```

Comment:
telemetrySamplingScore is a dynamically generated value based on operation id. This cannot be set as Sampling percentage after sampled in since this will affect the item count value in the analytics portal. So replaced this with testedPercentage which will have the correct sampling percentage.